### PR TITLE
Add WaveICA

### PR DESCRIPTION
**Pull request by maximskorik**
This PR adds WaveICA package to Bioconda. WaveICA is a batch-correction tool for large-scale metabolomics LC-MS data.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/r-recetox-waveica/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - $R -e "library('WaveICA')"
+
+about:
+  home: "{{ github }}"
+  license: MIT
+  summary: Removal of batch effects for large-scale untargeted metabolomics data based on wavelet analysis.
+
+extra:
+  maintainers:
+    - RECETOX/conda
```
Comment (martenson) [inline recipes/r-recetox-waveica/meta.yaml:56]: We've tested this and it sadly does not work, the teams are invisible/not pingable from outside of the organisation. These have to be actual personal accounts.

Comment (maximskorik) [reply recipes/r-recetox-waveica/meta.yaml]: I see. That's a pity. Updated the maintainers in af65299.

**Comment by maximskorik** [comment]
@BiocondaBot please add label

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
We need to add r-plsdepot to conda-forge. Currently it's getting pulled in from the deprecated R channel, which is going to be incompatible with a lot of things.

**Comment by dpryan79** [comment]
Xref: https://github.com/conda-forge/staged-recipes/pull/16900

**Comment by dpryan79** [comment]
r-plsdepot should be available in a couple hours.

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Comment by maximskorik** [comment]
Thank you, @dpryan79!

**Pull request merged**